README.md

```diff
@@ -66,11 +66,12 @@ for the submodule in the changes submitted to Docker.Powershell.
 
 ### Compiling with DotNet CLI
 To compile this project, you need the following:
-* A recent build of the [.NET Core SDK](https://github.com/dotnet/cli)
+* A recent build of the [.NET Core SDK](https://github.com/dotnet/cli/releases)
 * The .NET 4.6 SDK. You get this either by:
```

Comment:
I'm not sure what you mean.  I was clarifying by pointing toward the releases, which is where recent builds can be installed from instead of to the main page where you need to clone and compile the CLI before installing.  Isn't that clearer?